scripts/test_images.sh

```diff
@@ -85,3 +85,18 @@ if [ "${GET}" != "${VALUE}" ]; then
 fi
 
 echo "Succesfully tested etcd local image ${TAG}"
+
+for TARGET_ARCH in "amd64" "arm64" "ppc64le" "s390x"; do
+    if [ "${TARGET_ARCH}" != "amd64" ]; then
+        ARCH_TAG=v"${VERSION}"-"${TARGET_ARCH}"
+    else
+        ARCH_TAG=v"${VERSION}"
+    fi
+
+    IMG_ARCH=$(docker inspect --format '{{.Architecture}}' "${REPOSITARY}:${ARCH_TAG}")
+    if [ "${IMG_ARCH}" != "$TARGET_ARCH" ];then
+        echo "Incorrect docker image architecture"
+        exit 1
+    fi
+    echo "Correct Architecture ${ARCH_TAG}"
+done
```

Comment:
Two comments:
1. There is no suffix "-amd64" for amd64 arch in 3.4. Please see https://github.com/etcd-io/etcd/pull/15609#discussion_r1161497890
2. 3.4 release doesn't support `s390x`

Reply:
Conditional added for amd64 arch tag and s390x removed. Thanks! 😅